modules/nw-allocate-load-balancers-to-subnets-procedure.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// *networking/allocating-load-balancers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-allocating-load-balancers-to-subnets-procedure_{context}"]
+= Specifying AWS Subnets for OpenShift Ingress Load Balancers at Installation
+
+Perform the following steps to allocate API and Ingress load balancers to specific subnets.
```

Comment:
Maybe there is a reason to capitalize ingress (I'm not sure) but normally I wouldn't see it capitalized.